src/components/ProjectAvatar.vue

```diff
@@ -0,0 +1,61 @@
+<template>
```

Comment:
This file is almost a copy/paste from TeamAvatar so I guess we can refactor things to have a single Avatar.vue component which takes right props and is used in both places ie for teams and projects

Reply:
You're right. I'll create a more generic Avatar component!